src/pages/ExamsPage/components/ReviewExamAttemptButton.jsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+
+import {
+  Button, useToggle, ModalDialog, ActionRow,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Warning } from '@edx/paragon/icons';
+import { useModifyExamAttempt } from '../hooks';
+
+const ReviewExamAttemptButton = ({
+  username, examName, attemptId, suspicionLevel,
+}) => {
+  const [isOpen, open, close] = useToggle(false);
+  const modifyExamAttempt = useModifyExamAttempt();
+  const { formatMessage } = useIntl();
+
+  return (
+    <>
+      <div className="d-flex text-danger">
+        <Button variant="link" size="sm" className="text-danger" onClick={open}>
+          <Warning />
+          {formatMessage({
+            id: 'ReviewExamAttemptButton.exam_name',
+            defaultMessage: 'Second Review Required',
+            description: 'Table header for the table column with buttons to review exam attempts',
+          })}
+        </Button>
+      </div>
+      <ModalDialog
+        title="my dialog"
+        isOpen={isOpen}
+        onClose={close}
+        size="md"
+        variant="default"
+        hasCloseButton
+        isFullscreenOnMobile
+      >
+        <ModalDialog.Header>
+          <ModalDialog.Title>
+            {formatMessage({
+              id: 'ReviewExamAttemptButton.confirmation_modal_title',
+              defaultMessage: 'Please confirm your choice.',
+              description: 'Title header of the modal that appears to confirm the review of an exam attempt',
+            })}
+          </ModalDialog.Title>
+        </ModalDialog.Header>
+
+        <ModalDialog.Body>
+          {formatMessage(
```

Comment:
We should also add the submission_reason text to the modal along with some text describing where to go to look at the recording.

Overall it might make sense to break this down into multiple elements. Otherwise we're going to end up with one giant string that would be impossible to translate since it's mixed with backend data. A possible layout

```
<p>Are you sure...
<p>Proctoring recordings and details can be viewed at...
<ul>
  <li>username: username
  <li>exam name...
  <li>suspicion calculation...
  <li>submission reason...
</ul>